todo_app/data/trello_items.py

```diff
@@ -10,7 +10,7 @@ def base_params():
     return {'key': Config().api_key, 'token': Config().api_token}
 
 def prepareUrlAndParams(endpoint, params):
-    return f'{base_url}/{api_version}/{endpoint}/', base_params() | params
+    return f'{base_url}/{api_version}/{endpoint}/', {**base_params(), **params}
```

Comment:
I know there was info on how to include installing a specific py version but I was running out of time unfortunately so had to do this to be compatible with python 3.7 on the host